grunttasks/selectconfig.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (grunt) {
     console.log('Using configuration files', process.env.CONFIG_FILES);
 
     // `server` is a shortcut to the server configuration
-    var serverConfig = require('../server/lib/configuration').root();
+    var serverConfig = require('../server/lib/configuration').getProperties();
```

Comment:
I'm *amazed* this is the only line that needs to be updated with all of the major release bumps.